src/adapters/third-party-providers-service-fetcher.ts

```diff
@@ -11,7 +11,8 @@ type ThirdPartyProvidersServiceResponse = {
 
 const SERVICE_URL_DICTIONARY = {
   polygon: 'https://third-party-providers-resolver.decentraland.org',
-  mumbai: 'https://third-party-providers-resolver.decentraland.zone'
+  mumbai: 'https://third-party-providers-resolver.decentraland.zone',
```

Comment:
Do we still need mumbai here?

Reply:
at least for the moment 😞  
when everything works as expected using `amoy, ' I'll remove the deprecated testnets.